python/uncertainties.ipynb

```diff
@@ -411,24 +413,27 @@
    "source": [
     "import sympy\n",
     "\n",
+    "\n",
     "def error(f, err_vars=None):\n",
     "    from sympy import Symbol, latex\n",
+    "\n",
     "    s = 0\n",
     "    latex_names = dict()\n",
-    "    \n",
-    "    if err_vars == None:\n",
+    "\n",
+    "    if err_vars is None:\n",
```

Comment:
This one is important to teach the right way, the other changes are made to make the linter happy.